Algorithms.Tests/Numeric/Decomposition/SVDTests.cs

```diff
@@ -1,136 +1,135 @@
-using Algorithms.Numeric.Decomposition;
```

Comment:
Have you changed CRLF to LF or why is this changed?

Reply:
Yes, this is an IDE formatting